src/helm/benchmark/static/schema.yaml

```diff
@@ -1822,6 +1850,20 @@ run_groups:
       main_name: none
       main_split: none
 
+  - name: decoding_trust
```

Comment:
We should either use `decoding_trust` everywhere or `decodingtrust` everywhere, for consistency.

Reply:
Thanks! We fixed this naming dependency issue in e155c21baaa7c868a7984f454b6ee68dbfd76f26.